grid/p5js/grid-closure/index.md

```diff
@@ -0,0 +1,12 @@
+layout: p5js
+author: Edmundo Mejía Galindo
+author-url: https://github.com/edgalindo
+title: Grid
```

Comment:
Please use a more specific name The category is called gird. At least use grid-closure